client/src/pages/reports/Show.js

```diff
@@ -588,7 +589,14 @@ const BaseReportShow = props => {
               </Fieldset>
 
               <Fieldset title="Meeting attendees">
-                <AttendeesTable attendees={report.attendees} disabled />
+                <AttendeesTable
+                  attendees={report.attendees}
+                  disabled
+                  showSensitive={
+                    report.reportSensitiveInformation &&
+                    report.reportSensitiveInformation.text
```

Comment:
This same condition now appears twice in this render, better extract it into an aptly named local variable.